pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -1743,6 +1743,22 @@ func TestValidatePipelineParameterVariables_Failure(t *testing.T) {
 			Paths:   []string{"[0].matrix[b-param].value[0]"},
 		},
 		api: "alpha",
+	}, {
+		name: "invalid task use duplicate parameters",
+		tasks: []PipelineTask{{
+			Name:    "foo-task",
+			TaskRef: &TaskRef{Name: "foo-task"},
+			Params: []Param{{
+				Name: "duplicate-param", Value: ParamValue{Type: ParamTypeString, StringVal: "val1"},
+			}, {
+				Name: "duplicate-param", Value: ParamValue{Type: ParamTypeString, StringVal: "val2"},
+			}},
+		}},
+		expectedError: apis.FieldError{
+			Message: `expected exactly one, got both`,
+			Paths:   []string{"[0][1].name"},
+		},
+		api: "alpha",
```

Comment:
Is this issue only happening with the alpha flag? If not, please drop setting this flag.

Reply:
you are right, it's not only happening with the alpha flag, i've dropped this flag

thanks @pritidesai